config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -91,3 +91,28 @@ periodics:
     testgrid-num-failures-to-alert: '1'
     testgrid-dashboards: sig-security-snyk-scan
     description: Run snyk scan on k/k master periodically
+- name: auto-refreshing-official-cve-feed
+  interval: 2h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: sig-security
+    base_ref: main
+  labels:
+    preset-service-account: "true"
+  spec:
+    serviceAccountName: k8s-cve-feed
+    containers:
+    - image: python3.7
+      command:
+      - cd sig-security-tooling/cve-feed/hack/ && chmod +x fetch-cve-feed.sh && ./fetch-cve-feed.sh
```

Comment:
```suggestion
      - cd sig-security-tooling/cve-feed/hack/ && ./fetch-cve-feed.sh
```

`chmod` should be done at the repo level. git will store the information in the commit. There is no need to rerun this command everytime the job is run.

Reply:
Updated